app/models/mpi_data.rb

```diff
@@ -137,8 +137,8 @@ def error
   end
 
   # @return [MPI::Responses::FindProfileResponse] the response returned from MVI
-  def mvi_response
-    @mvi_response ||= response_from_redis_or_service
+  def mvi_response(user_key: user_identity.uuid)
```

Comment:
Are we sure there's no one else in `vets-api` currently calling this method and that this won't have an impact on them?